### DISCUSSION TITLE
where I can find vllm_flash_attn's source code?

### DISCUSSION THREAD

**Discussion by huizhanyi** [vllm-project/vllm#10033] [Q&A]
I want to debug the vllm_flash_attn code, but the release tar has no debug info. So I want to compile it, but I cannot find the source code , and do not know how to build it with source code. Need help.

**Comment by jeejeelee** [comment]
See：https://github.com/vllm-project/flash-attention and https://github.com/vllm-project/vllm/blob/main/CMakeLists.txt#L507

**Comment by huizhanyi** [comment]
thanks，it is here
https://github.com/vllm-project/flash-attention.git
